dev-docs/modules/userId.md

```diff
@@ -254,6 +254,7 @@ Bidders that want to support the User ID module in Prebid.js need to update thei
 | Quantcast ID | n/a | quantcastId | quantcast.com | "1111" |
 | Tapad ID | Tapad | tapadId | tapad.com | "1111" |
 | Teads ID | Teads | teadsId | teads.com | "1111" |
+| TNC ID | The Newco | tncId | thenewco.it | "1111" |
```

Comment:
This is not going be necessary thanks to #5673 , which generates this automatically.